registration-processor/pre-processor/registration-processor-cmd-validator-stage/src/main/java/io/mosip/registration/processor/stages/app/CMDValidationProcessor.java

```diff
@@ -1,259 +1,260 @@
+import io.mosip.registration.processor.core.exception.util.PlatformErrorMessages;
+import io.mosip.registration.processor.core.exception.util.PlatformSuccessMessages;
+import io.mosip.registration.processor.core.logger.LogDescription;
+import io.mosip.registration.processor.core.logger.RegProcessorLogger;
+import io.mosip.registration.processor.core.packet.dto.RegOsiDto;
+import io.mosip.registration.processor.core.status.util.StatusUtil;
+import io.mosip.registration.processor.core.status.util.TrimExceptionMessage;
+import io.mosip.registration.processor.core.util.RegistrationExceptionMapperUtil;
+import io.mosip.registration.processor.packet.storage.exception.ParsingException;
+import io.mosip.registration.processor.packet.storage.utils.OSIUtils;
+import io.mosip.registration.processor.packet.storage.utils.PriorityBasedPacketManagerService;
+import io.mosip.registration.processor.rest.client.audit.builder.AuditLogRequestBuilder;
+import io.mosip.registration.processor.stages.cmdvalidator.CenterValidator;
+import io.mosip.registration.processor.stages.cmdvalidator.DeviceValidator;
+import io.mosip.registration.processor.stages.cmdvalidator.MachineValidator;
+import io.mosip.registration.processor.status.code.RegistrationStatusCode;
+import io.mosip.registration.processor.status.dto.InternalRegistrationStatusDto;
+import io.mosip.registration.processor.status.dto.RegistrationStatusDto;
+import io.mosip.registration.processor.status.exception.TablenotAccessibleException;
+import io.mosip.registration.processor.status.service.RegistrationStatusService;
+
+@Service
+@Transactional
+public class CMDValidationProcessor {
+
+	private static Logger regProcLogger = RegProcessorLogger.getLogger(CMDValidationProcessor.class);
+
+	private TrimExceptionMessage trimExpMessage = new TrimExceptionMessage();
+
+	private static final String USER = "MOSIP_SYSTEM";
+
+	public static final String GLOBAL_CONFIG_TRUE_VALUE = "Y";
+
+	@Autowired
+	private RegistrationStatusService<String, InternalRegistrationStatusDto, RegistrationStatusDto> registrationStatusService;
+
+	@Autowired
+	private AuditLogRequestBuilder auditLogRequestBuilder;
+
+	@Autowired
+	private RegistrationExceptionMapperUtil registrationStatusMapperUtil;
+
+	@Autowired
+	private PriorityBasedPacketManagerService packetManagerService;
+
+	@Autowired
+	private CenterValidator centerValidator;
+
+	@Autowired
+	private MachineValidator machineValidator;
+
+	@Autowired
+	private DeviceValidator deviceValidator;
+
+	@Autowired
+	private OSIUtils osiUtils;
+
+	@Value("${mosip.registration.gps_device_enable_flag}")
+	private String gpsEnable;
+
+	@Value("${mosip.primary-language}")
+	private String primaryLanguagecode;
+
+	@Value("#{'${mosip.regproc.cmd.center.validation.processes:CORRECTION,NEW,UPDATE,LOST,BIOMETRIC_CORRECTION,DOCUMENT_CORRECTION,DEMOGRAPHIC_CORRECTION}'.split(',')}")
+	private List<String> centerSubProcesses;

```

Comment:
Please change the property to mosip.regproc.cmd-validator.center-validation.processes:NEW,UPDATE,LOST,BIOMETRIC_CORRECTION

We can only have the processes that we are going to have as samples

Also please change the variable name to centerValidationProcessList (This is not only sub process)

Follow this config and variable naming conventation for other 2 properties also

Reply:
Property name is not maching with what I have commented above

Reply:
I have changed the properties as requested